erts/preloaded/src/erlang.erl

```diff
@@ -8409,8 +7873,8 @@ Valid `InfoTuple`s with corresponding `Item`s:
   `GCInfo` is a list containing miscellaneous detailed information about
   garbage collection for this process. The content of `GCInfo` can be changed
   without prior notice. For details about the meaning of each item, see
-  [`gc_minor_start`](#gc_minor_start) in
-  [`erlang:trace/3`](`trace/3`).
+  [`gc_minor_start`](`e:kernel:trace#gc_minor_start`) in
```

Comment:
```suggestion
  [`gc_minor_start`](`m:trace#gc_minor_start`) in
```

Reply:
So `e:kernel` is not needed when referring from `erts` as module `trace` is unique anyway?

Reply:
correct. The `e` prefix is for when you reference anything that is not a module in a different application, that is user's guides, release notes etc etc.